data/coalition/coalition news.txt

```diff
@@ -428,6 +429,7 @@ news "coalition games crowd"
 			"Coalition Games crowds"
 	message
 		word
-			`The lines of people heading to the Coalition Games seem to have stopped moving abruptly. Upon further investigation, you see some young Kimek have fallen on their backs, being helped back to their feet one by one by Heliarch agents and others from the crowd.`
-			`You arrive to a particularly crowded moment of the spaceport. The lines of people waiting for admission into the Games have now started looping around, heading inside some shops and restaurants, and even a bit into the docking area. Battalions of Heliarch agents work as they can to bring everyone back into orderly lines.`
-			`From the moment you arrive at the spaceport, Heliarch agents start showing you where you want to go if you're headed for the Coalition Games, which line leads to which sporting event, where to go to if you need some general assistance from the Heliarchy, and give you a schedule of the Games for today.`
+			"The lines of people heading to the Coalition Games seem to have stopped moving abruptly. Upon further investigation, you see some young Kimek have fallen on their backs, being helped back to their feet one by one by Heliarch agents and others from the crowd."
+			"You arrive at a particularly crowded part of the spaceport. The lines of people waiting for admission into the Games have now started looping around, heading inside some shops and restaurants, and even a bit into the docking area. Battalions of Heliarch agents work as they can to bring everyone back into orderly lines."
+			"Entering the tremendously crowded spaceport, you notice a Saryd pastry shop opening up for business, and decided to head there for a snack. When you finally get to the door, however, they're just done closing. It seems the masses heading to the Coalition Games bought all their products as the line was moving."
```

Comment:
```suggestion
			"Entering the tremendously crowded spaceport, you notice a Saryd pastry shop opening up for business, and decide to head there for a snack. When you finally get to the door, however, they're just done closing. It seems the masses heading to the Coalition Games bought all their products as the line was moving."
```
This is quite different from other news in its style; most news are about First Last observing or about people talking to them, while in this one First Last actually does something.

Also, I'm not sure if `notice` and `decide` should be present or past, but they should probably be in the same tense.